website/docs/components/form/textarea/partials/accessibility/support.md

```diff
@@ -0,0 +1,4 @@
+---
+
+## Support
+If any accessibility issues have been found within this component, please let us know by [submitting an issue](https://github.com/hashicorp/design-system/issues/new/choose).
```

Comment:
```suggestion
If any accessibility issues have been found within this component, let us know by [submitting an issue](https://github.com/hashicorp/design-system/issues/new/choose).
```

Reply:
in this case we repeat this exact some content in many places, so changing it here is minimal value but I can accept it nonetheless. when we replace this with a reusuable, global component we should be mindful of this language there as well.

Reply:
Agreed, let's keep it consistent and leave the "please" in here for now.